<launch>
    <include file="$(find sub8_trajectory_generator)/launch/tgen_params.launch" />
	<node pkg="sub8_trajectory_generator" type="trajectory_generator" name="sub8_trajectory_generator" output="screen"/>
</launch>